flask_apispec/wrapper.py

```diff
@@ -43,8 +43,11 @@ def call_view(self, *args, **kwargs):
                 parsed = parser.parse(schema, locations=option['kwargs']['locations'])
                 if getattr(schema, 'many', False):
                     args += tuple(parsed)
-                else:
+                elif getattr(parsed, 'update', False):
```

Comment:
There is risk of a false-positive here, e.g. model objects with an `update` method, so I think type-checking is preferable to duck-typing in this case.

I'll make this change myself.